### PR TITLE
initial src for Oracle AI For Healthcare (Open Source Version) Livelabs Workshop

### PR DESCRIPTION
**Pull request by paulparkinson**
initial src for Oracle AI For Healthcare (Open Source Version) Livelabs Workshop

### CODE REVIEW

File: flutter-frontend/devenv.sh
Comment (jasperan) [inline flutter-frontend/devenv.sh:1]: @paulparkinson be mindful of passwords in comments - they are considered a security vulnerability

File: springboot-backend/createSecretFromWallet.sh
Comment (jasperan) [inline springboot-backend/createSecretFromWallet.sh:1]: Also, be mindful of credentials here in line 2 as well @paulparkinson

File: springboot-backend/devenv.sh
Comment (jasperan) [inline springboot-backend/devenv.sh:1]: Here as well, watch out with your API keys! @paulparkinson 

I recommend loading them from a JSON file or metadata file, or adding this file in .gitignore to ignore it

**Review by jasperan — CHANGES_REQUESTED** [review]
Hi @paulparkinson, I added some notes for you to review in 3 files

**Review by paulparkinson — COMMENTED** [review]
@jasperan thanks! took care of them

**Review by jasperan — APPROVED** [review]

**Pull request merged**